### PR TITLE
[SDESK-4062] Fix scroll in Media Contacts

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: scripts/apps/contacts/directives/ContactList.ts [hunk 1]
```diff
@@ -148,9 +149,9 @@ export function ContactList(
                 return element.scrollTop + element.offsetHeight + 200 >= element.scrollHeight;
             }
 
-            elem.on('keydown', listComponent.handleKey);
+            scrollElem.on('keydown', listComponent.handleKey);
 
-            elem.on('scroll', handleScroll);
+            scrollElem.on('scroll', handleScroll);
```
Comment (petrjasek) [inline scripts/apps/contacts/directives/ContactList.ts]: I think you should also call `scrollElem.off();` on `$destroy` event

Comment (marwoodandrew) [reply scripts/apps/contacts/directives/ContactList.ts]: OK

Comment (marwoodandrew) [reply scripts/apps/contacts/directives/ContactList.ts]: Done

**Comment by MarkLark86** [comment]
@petrjasek This bug fix is required in 1.29 & 1.30 as well (as you cannot load more items once reaching the bottom of the list).

**Pull request merged**